src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -83,6 +83,11 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack
 		) {
+			if( type is IErrorTypeSymbol  || type == null ) {
+				// type can't be resolved; the project is either not configured correctly, or there's a typo
+				return false;

```

Comment:
I'm worried about us marking unsafe code as safe and never knowing about it/creating headaches later when we realize the mistake but need to keep CI working. I think defaulting to unsafe makes the most sense.